### DISCUSSION TITLE
Option group - function for select

### DISCUSSION THREAD

**Discussion by labs20** [quasarframework/quasar#12552] [General - Components / Directives / etc]
Hi. I'm in need to do the basic set of operations on an option-group via code, like select an specific option, select the next one, select previous and clear, but there is no functions for that.

I've tryed to manipulate the model-value but the visuals are not ok (focus on "selected" option is incorrect).

I need to do that only because I need arrow keys to navigate the options, so I'd like to ask if there is a way to implement such functions, or extend the keyboard navigation to the arrow keys.

P.s: IMHO, Quasar is still not "programing friendly" as it lacks most of the apis for controling the obvious features of many components. There are plans to change that?

Thank you.

**Comment by pdanpdan** [comment]
Make a codepen with what you are doing - from my POV it's not clear what you need.
